### PR TITLE
ci(core): Reduce memory usage in tests (part-2) (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**
This also gets rid of `Db.collection`, which was another source of circular dependencies.

### CODE REVIEW

File: packages/cli/src/commands/export/credentials.ts [hunk 1]
```diff
@@ -110,7 +111,8 @@ export class ExportCredentialsCommand extends BaseCommand {
 			findQuery.id = flags.id;
 		}
 
-		const credentials: ICredentialsDb[] = await Db.collections.Credentials.findBy(findQuery);
+		const credentials: ICredentialsDb[] =
```
Comment (ivov) [inline packages/cli/src/commands/export/credentials.ts:114]: We might not need the typing here.

File: packages/cli/src/Db.ts [hunk 2]
```diff
@@ -165,32 +140,6 @@ export async function init(testConnectionOptions?: ConnectionOptions): Promise<v
...
-	collections.WorkflowTagMapping = Container.get(WorkflowTagMappingRepository);
-
-	/**
-	 * @important Do not remove these collections until cloud hooks are backwards compatible.
-	 */
-	collections.Role = Container.get(RoleRepository);
-	collections.User = Container.get(UserRepository);
-	collections.Settings = Container.get(SettingsRepository);
-	collections.Credentials = Container.get(CredentialsRepository);
-	collections.Workflow = Container.get(WorkflowRepository);
```
[the remark this answers is not in the document]
Comment (netroy) [reply packages/cli/src/Db.ts:193]: no. the only place we need the original `dbCollections` is the `this` arg for external hooks. and that is now only passing in the repositories that are actually used in the cloud hooks.

File: packages/cli/src/commands/export/credentials.ts (hunk 1 above)
Comment (netroy) [reply packages/cli/src/commands/export/credentials.ts:114]: unfortunately we still do. `ICredentialsDb` and `CredentialsEntity` are still two different types.

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.17.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.17.0)

**Pull request merged**